src/main/kotlin/no/nav/klage/search/api/controller/KlagebehandlingListController.kt

```diff
@@ -80,6 +82,39 @@ class KlagebehandlingListController(
         )
     }
 
+    @ApiOperation(
+        value = "Hent saksbehandlere i gitt enhet",
+        notes = "Henter alle saksbehandlere fra aktive saker i gitt enhet."
+    )
+    @GetMapping("/{navIdent}/enheter/{enhet}/saksbehandlere", produces = ["application/json"])
+    fun getSaksbehandlereForEnhet(
+        @ApiParam(value = "NavIdent til en ansatt")
+        @PathVariable navIdent: String,
```

Comment:
Jeg vet ikke om jeg helt forstår hva denne er her for? Du validerer den noen linjer lenger ned, men bruker du den til noe?

Reply:
Innlogget saksbehandler valideres mot `navIdent` som er i ressurs-pathen. Innlogget saksbehandler brukes så for å sjekke tilgang til `enhet`. Men vi kunne alltid utledet `navIdent` her basert på token, men tror det gir mening å la den ligge som del av path. Tror ikke det skader iaf. Men vi kunne vurdert noe som `/me/enheter/id/etc`.

Reply:
Det brukes vel her kun for å sjekke tilgang, og da tenker jeg det egentlig er greit å ta det fra tokenet. Det er ikke sånn at du og jeg skal få ulike lister basert på identen vår, så identen har ingen funksjonell betydning. AFAIK.